examples/siro_sandbox/app_states/app_state_fetch.py

```diff
@@ -99,7 +128,37 @@ def __init__(self, sandbox_service, gui_agent_ctrl, robot_agent_ctrl):
     def _is_remote_active(self):
         return self._is_remote_active_toggle
 
-    def on_environment_reset(self, episode_recorder_dict):
+    def _make_robot_face_human(self, human_pos, robot_pos):
+        relative_target = human_pos - robot_pos
+        angle_to_object = get_angle_to_pos(relative_target)
+        self.get_sim().get_agent_data(
+            1 - self.get_gui_controlled_agent_index()
+        ).articulated_agent.base_rot = angle_to_object
+
+    def _make_robot_near_human(self, human_pos):
+        """Constrain the robot location to be near human pos"""
+        robot_pos = self.get_sim().pathfinder.get_random_navigable_point_near(
+            human_pos,
+            ROBOT_SPAWN_NEAR_HUMAN_DIS,
+            island_index=self.get_sim()._largest_island_idx,
+        )
+
+        # If the robot_pos is not NaN
+        if not np.isnan(robot_pos[0]):
```

Comment:
Should we assert here? Don't we want to know if this fails?

Reply:
`assert` here is risky since the whole app crashes. The logic here is that we did our best to find a good robot pos.